src/components-styled/landelijke-toelichting.tsx

```diff
@@ -0,0 +1,54 @@
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import ExternalLinkIcon from '~/assets/external-link.svg';
+import siteText from '~/locale';
+import theme from '~/style/theme';
+import { Box } from './base';
+import { ExternalLink } from './external-link';
+import { Tile } from './tile';
+import { Heading, Text } from './typography';
+
+export function LandelijkeToelichting() {
+  return (
+    <Tile
+      css={css({
+        bg: theme.colors.contextualContent,
+        flexDirection: [null, 'row'],
+      })}
+    >
+      <Box flex="1 1 25%">
+        <img src="images/toelichting-afbeelding.png" />
+      </Box>
+      <Box flex="1 1 75%" pl={[null, 4]}>
+        <Text mt={0} color="gray">
+          {siteText.notificatie.subtitel}
+          {' \u2022 '}
```

Comment:
I think this is more readable:
```suggestion
          {' • '}
```